src/components/MangaControl.tsx

```diff
@@ -35,83 +49,115 @@ const MangaControl: React.FunctionComponent<Props> = (props: Props) => {
   const changePage = (event: React.ChangeEvent<HTMLSelectElement>) => {
     updatePage(+event.currentTarget.value);
   };
+  const nextChapter = (next = true) => {

```

Comment:
why is this called nextChapter if you can also use them to go back chapters.

prob just seperate it out into two functions and u dont need to pass in a bool anymore